.github/workflows/ci-main.yml

```diff
@@ -1,16 +1,25 @@
 on:
   pull_request:
     branches:
-      - 'main'
+      - 'main' # excluding main
```

Comment:
Doesn't mean "only main" instead of "excluding main"?

Reply:
Yes of course, in any case this comment is from copying the `dev` and only temporary.